### PR TITLE
Remove JavaScript from footer

### PR DESCRIPTION
**Pull request by gaul**
Addresses an error while building Javadoc.

### CODE REVIEW

**Comment by gaul** [comment]
Blocks 2.1.2-rc1.

File: project/pom.xml [hunk 1]
```diff
@@ -1120,19 +1120,6 @@
             <links>
               <link>http://download.oracle.com/javase/6/docs/api/</link>
             </links>
-            <footer><![CDATA[
-            <!-- Google Analytics -->
```
Comment (demobox) [inline project/pom.xml]: Out of curiosity: do we know whether there was ever a GA account associated with this?

Comment (nacx) [reply project/pom.xml]: If we have it we never used it. I think Everett added this?

Comment (gaul) [reply project/pom.xml]: Looks like Adrian in 0e07b7236fb594146bb8c3cd485004716f2897c0.

**Pull request merged**